src/main/java/org/gridsuite/directory/server/repository/DirectoryElementEntity.java

```diff
@@ -38,8 +40,8 @@
     @Column(name = "type", length = 30, nullable = false)
     private String type;
 
-    @Column(name = "isPrivate", nullable = false)
-    private boolean isPrivate;
+    @Column(name = "isPrivate")
+    private Boolean isPrivate;
```

Comment:
should be private and not isPrivate, this is why you end up with ugly get getIsPrivate

Reply:
it already was isPrivate I didn't change that i changed the type so it switched to getIsPrivate (it works well with boolean but not Boolean wonder why)

Reply:
also, good luck naming a variable "private"